### PR TITLE
Content/topic errors amend

### PR DESCRIPTION
**Pull request by linuxmonk**
* Add Header / beginning slide
* Changed (continued) to (1/2), (2/2) convention
* Updated "Errors in Go (1/2)" slide to make it simpler and add a point about naming convention of error variables
* Modified "Error Handling" slide

Slides: https://pr101-dot-gotraining-testing.appspot.com/gocourse-2019-05-melbourne.slide#70

### CODE REVIEW

File: content/gocourse-2019-05-melbourne.slide [hunk 1]
```diff
@@ -874,25 +876,18 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
 	}
 - Check errors explicitly
 
-* Errors in Go (continued)
+* Errors in Go (1/2)
 
-- The [[https://golang.org/pkg/errors/][`errors`]] package has functions for creating new error values
+- Create error values
 - Using [[https://golang.org/src/errors/errors.go?s=293:320#L1][`errors.New()`]]:
```
Comment (juliaogris) [inline content/gocourse-2019-05-melbourne.slide]: Fold into one line:  
Create error values using `errors.New()`

File: content/gocourse-2019-05-melbourne.slide [hunk 2]
```diff
@@ -908,20 +903,18 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
...
 
-- Check for the presence of an error by [[https://goplay.space/#40Ho_HWiYrb][comparing with nil]]
-- [[https://goplay.space/#T2RkoxxpuUV][Comparing error values]]
-- Errors are values so they can be [[https://goplay.space/#F_EG5jM_fZ6][programmed]]
-- [[https://goplay.space/#bTOIkCkGssE][Handle errors gracefully]]
-- Check errors using [[https://goplay.space/#ISKTAUFOyjk][type assertions]]
+- Check for the [[https://goplay.space/#X0qqmcrO6om][presence of an error]]
+- Comparing [[https://goplay.space/#HiLw8Y_D_6T][error values]]
+- Checking error types using [[https://goplay.space/#CII8be9-TqW][type assertions]]
+- Errors are values so they can be [[https://goplay.space/#I6UJNHe6C7v][programmed]]
```
Comment (juliaogris) [inline content/gocourse-2019-05-melbourne.slide]: IMO less words are easier to parse on slides and more consistent with existing one.
I'd use as full links with nothing else around it:
- Check for error
- Compare error values
- Compare error types
- Program with error

File: content/gocourse-2019-05-melbourne.slide [hunk 3]
```diff
@@ -908,20 +903,18 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
...
+- Error handling patterns in the standard library
```
Comment (juliaogris) [inline content/gocourse-2019-05-melbourne.slide]: I'd remove this line - it doesn't tell us anything when reading the slides.

**Review by juliaogris — COMMENTED** [review]
a few minor things.

**Comment by linuxmonk** [comment]
Resolved all the changes. Thanks.

**Review by juliaogris — APPROVED** [review]
LGTM

**Pull request merged**